sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveDirCommand.scala

```diff
@@ -90,6 +90,15 @@ case class InsertIntoHiveDirCommand(
         val dfs = qualifiedPath.getFileSystem(hadoopConf)
         (qualifiedPath, dfs)
       }
+
+    if (fs.exists(writeToPath) && fs.isFile(writeToPath)) {
```

Comment:
We need `fs.exists(writeToPath)` here? Does `fs.isFile(writeToPath)` return false if the file does not exist?

Reply:
Yes, if the target path does not exist, it will return false.